index.html

```diff
@@ -1,173 +1,352 @@
-                Nullam vestibulum orci a sapien imperdiet maximus. 
-                Suspendisse vel ex eget mauris efficitur finibus eu a enim. 
-                Aenean tellus neque, consectetur sit amet lacus a, hendrerit ornare urna. 
-                Fusce venenatis lorem vitae augue suscipit, eu tempor massa vulputate. 
-                Praesent iaculis velit libero, vel faucibus sem pulvinar vel. 
-                Curabitur viverra nisl dui, a venenatis felis pellentesque id. 
-                Donec non lacus diam.
-            </p>
+        <div class="div_Navigation">
+            <a class="anchorNav" id="anchorID1" href="#anchorID2">Ugrás az oldal aljára</a>
+            <a class="anchorNav" id="anchorIDRegistration" href="#idH1Registration">Regisztráció</a>
+            <a class="anchorNav" id="anchorIDSendArticle" href="#idH1SendArcticle">Cikk küldése</a>
+        </div>
 
-        <h2>Cimsor 2 - Lorem ipsum hosszu</h2>
-            <p>
-                Lorem ipsum dolor sit amet, consectetuer adipiscing elit
-                Aenean commodo ligula eget dolor
-                Aenean massa
-                Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus
-                Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem
-                Nulla consequat massa quis enim
-                Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu
-                In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo
-                Nullam dictum felis eu pede mollis pretium
-                Integer tincidunt
-                Cras dapibus
-            </p>      
-            <p>
-                Vivamus elementum semper nisi
-                Aenean vulputate eleifend tellus
-                Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac, enim
-                Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus
-                Phasellus viverra nulla ut metus varius laoreet
-                Quisque rutrum
-                Aenean imperdiet
-                Etiam ultricies nisi vel augue
-                Curabitur ullamcorper ultricies nisi
-                Nam eget dui
-                Etiam rhoncus
-                Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero, sit amet adipiscing sem neque sed ipsum
-                Nam quam nunc, blandit vel, luctus pulvinar, hendrerit id, lorem
-                Maecenas nec odio et ante tincidunt tempus
-                Donec vitae sapien ut libero venenatis faucibus
-                Nullam quis ante
-                Etiam sit amet orci eget eros faucibus tincidunt
-                Duis leo
-                Sed fringilla mauris sit amet nibh
-                Donec sodales sagittis magna
-                Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,
-            </p>
+        <div style="
+                width: 100vw;
```

Comment:
Inline style abszolút ellenjavallt, egy ilyen PR-t munkahelyen nem fogadnék el.

Reply:
Rendben, modositottam (commit 9e081b9). 

Azert hasznaltam, mert ezek csak egy helyen fordultak elö, es gondoltam megmutatom a hf-ben, hogy az inline style-t is ismerem... :D